### ISSUE TITLE
In some states, cities aren't in counties

### ISSUE THREAD

**Issue by astrowonk** [balsama/us_counties_data#1]
e.g. Virginia, by only including the 95 counties and not the cities that don't exist inside a county, but are independent of any county, you're missing a lot of the population.

**Comment by balsama** [comment]
I'd love to be able to include more complete data. Do you have any insight into how the cities you mention are typically included in lists like these? do they have FIPS codes?

**Comment by astrowonk** [comment]
Yes, you can see [FIPS for cities and counties at the census](https://www.census.gov/geographies/reference-files/2018/demo/popest/2018-fips.html) (I believe "place" fips codes, 5 digits, do the job.)

The county population tables I keep finding though [tend to just have names](https://www.census.gov/data/tables/time-series/demo/popest/2010s-counties-total.html), so I ended up using [the data sets here](https://github.com/prairie-guy/2019-State-and-County-Population-with-FIPS-key), though I had to clean up the UTF-8 encoding myself.

I wish the census just hosted a population table with FIPS and by county/city.
